### ISSUE TITLE
Coq Platform Readme.md does not point to latest release

### ISSUE THREAD

**Issue by andrew-appel** [coq/platform#322]
When I want to install the Coq Platform, I go to the main README.md for instructions, but unfortunately it does not point me to the latest release.

Right now the latest release is 2022.09.1, but the most recent release in the README is 2022.09.  

The simplest way to fix this is to delete from the README the long list of specific releases, and say instead:  To get the latest release, or any previous release, visit https://github.com/coq/platform/releases

**Comment by MSoegtropIMC** [comment]
Sorry, I can't quite follow you. As far as I can see the main ReadMe has a list of supported Coq versions - which is something else than a Coq Platform release. The latest release of Coq Platform supports all these versions of Coq.

The platform specific ReadMe files have a link to https://github.com/coq/platform/releases as you suggest.

Where do you see a link to an old release in one of the readme files?

**Comment by andrew-appel** [comment]
This file:
https://github.com/coq/platform/blob/main/README.md
lists 8 different specific Platform releases, none of which is the newest one.

Instead of listing specific releases in that README.md, I suggest pointing to the general "releases" page that's automatically up to date.

**Comment by MSoegtropIMC** [comment]
No, this is a list of the Coq versions and package picks supported by the **latest** version of Coq Platform.

This list is accurate and up to date (e.g. it names Coq 8.16.1, not 8.16.0). The package pick did not change (short of bug fixes like 8.16.0 -> 8.16.1 and compatible changes) since September.

For the top two entries in the list the release provides installers. All other versions can be installed with the scripts.

The point of this is to ensure that one can still build old versions of Coq, e.g for artefact reproduction and for porting. This is not trivial, e.g. there were issues in the OCaml originally used for 8.13 and earlier which make the original 8.13 builds impossible to run on recent Linux. By supporting old Coq versions in latest Coq Platform versions we can test that all this actually works in CI. Specific requests for bug fix back ports in OCaml came from this CI.

**Comment by MSoegtropIMC** [comment]
P.S.: since it is obviously unclear what this list is, we should improve the documentation here. Do you have suggestions?

**Comment by andrew-appel** [comment]
The main web page for Coq Platform, which is basically the README.md under discussion, should say as prominently and early as possible,

**To install the latest Coq Platform release, [look here](https://github.com/coq/platform/releases/latest).**

**Comment by MSoegtropIMC** [comment]
OK, makes sense. We have this via an indirection to the platform specific pages, but for people who know this already it is indeed not ideal. I will change it as you suggest.

**Comment by MSoegtropIMC** [comment]
Triage note: should be fixed, but we will retest.
